interfaces/main.py

```
@@ -302,11 +302,18 @@ def _stop_autopilot_daemon_thread():
             _daemon_process.join(timeout=2)
         else:
             logger.info("✅ 守护进程已成功停止")
-    
+
     _daemon_process = None
     _daemon_stop_event = None
 
 
+def restart_autopilot_daemon():
+    """重启守护进程以拾取新的 LLM / 嵌入配置（跨进程 env 不可共享，必须重启）。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)